web/templates/email/daily_digest.text.eex

```diff
@@ -1,35 +1,31 @@
-<%= gettext("# Daily digest of new Interests and Announcements\n") %>
+<%= gettext("Daily digest of new Interests and Announcements\n") %>
+=================
 
 <%= unless Enum.empty?(@interests) do %>
 <%= gettext("Recently added interests") %>
 -----------------
 
 <%= for interest <- @interests do %>
-  #<%= interest.name %> - <%= interest_url(Constable.Endpoint, :show, interest) %>
+ - #<%= interest.name %> (<%= interest_url(Constable.Endpoint, :show, interest) %>)
 <% end %>
-
 <% end %>
 <%= unless Enum.empty?(@announcements) do %>
-<%= gettext("Recently added announcements\n") %>
+<%= gettext("Recently added announcements") %>
```

Comment:
How's `gettext` work in Elixir? Does changing this string affect something else?